plugins/phile/simpleFileDataPersistence/Classes/Persistence/SimpleFileDataPersistence.php

```diff
@@ -24,6 +24,9 @@ class SimpleFileDataPersistence implements PersistenceInterface {
 	 */
 	public function __construct() {
 		$this->dataDirectory = LIB_DIR . 'datastorage/';
+		if (!is_dir($this->dataDirectory)) {
+			mkdir($this->dataDirectory, 0755);
```

Comment:
> There's a file datastorage/.htaccess which isn't created.
@Schlaefer means, that after creating "deleted" folder `datastorage/`,
Phile should also create "deleted" `datastorage/.htaccess` file